Rakefile

```diff
@@ -13,7 +13,7 @@ end
 namespace "podspec" do
   desc "Run lint for podspec"
   task :lint do
-    run "bundle exec pod lib lint"
+    run "bundle exec pod lib lint --platforms=ios,tvos,macos"
```

Comment:
We need to skip visionos for now as the CI does not yet have visionOS simulators installed. We did the same for Nimble and need to revisit this once available.